framework/sandstone.cpp

```diff
@@ -1336,6 +1338,54 @@ static std::string cpu_features_to_string(uint64_t f)
     return result;
 }
 
+static uint64_t cpu_features_to_cpuid(std::string_view f_str)
+{
+    auto find_feature = [](std::string_view what) -> std::optional<uint64_t> {
+        for (size_t i = 0; i < std::size(x86_locators); i++) {
+            std::string_view checked_feature = std::string_view(features_string + features_indices[i] + 1);
+
+            if (checked_feature == what) {
+                return UINT64_C(1) << i;
```

Comment:
We're likely going to 128 bits soon, so please be future-proof here. Add before the lambda:
```c++
using Int = decltype(_compilerCpuFeatures);
```
Here use `Int(1) << i`

Reply:
Or modify framework/scripts/x86simd_generate.pl  to emit a `typedef` with the type we've used (remember the output is also used by C code).

Reply:
I like more modifying `x86simid_generate.pl`